opensrp-app/src/test/java/org/smartregister/util/JsonFormUtilsTest.java

```diff
@@ -1504,4 +1504,19 @@ public void addObservationAddsGpsObservationsToEvent() throws Exception {
             }
         }
     }
+
+    @Test
+    public void testCreateObservationForNativeRadioShouldAddKeyValuePairsToObservation() throws Exception {
```

Comment:
Also verify that the original value (which is the key in the case of radio buttons) is saved as well in the obs.

Reply:
done